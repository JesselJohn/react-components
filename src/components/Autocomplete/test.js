import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import TestComponent from "./testcomponent";

describe("Root Component Tests", () => {
  beforeAll(() => shallow(React.createElement(TestComponent)));

  it("To Match SnapShot", () => {
    expect(shallowToJson(TestComponent)).toMatchSnapshot();
  });
});
