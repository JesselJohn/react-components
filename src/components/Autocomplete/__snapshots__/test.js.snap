// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Autocomplete Root Component Tests To Match SnapShot when showSearch is false  1`] = `null`;

exports[`Root Component Tests To Match SnapShot 1`] = `null`;
